src/bivariate.jl

```diff
@@ -19,7 +19,7 @@ mutable struct BivariateKDE{Rx<:AbstractRange,Ry<:AbstractRange} <: AbstractKDE
     "Second coordinate of gridpoints for evaluating the density."
     y::Ry
     "Kernel density at corresponding gridpoints `Tuple.(x, permutedims(y))`."
-    density::Matrix{Float64}
+    density::AbstractMatrix{}
```

Comment:
relax type casting for future compatibility.. will make a PR to `Tracker.jl` soon